.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>RepoSense</name>
+	<comment>Project RepoSense created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+</projectDescription>
```

Comment:
Ehh, sorry if I missed anything, but may I know what is this file for?
If i'm not wrong its the Eclipse IDE project description file right? This shouldn't be inside our repo 😱 .

Reply:
Apology for the oversight. Must have accidentally commited it when activating Java debugging